docs/updater.md

```diff
@@ -125,3 +125,21 @@ If you can't fetch updates, perform a few checks from your XOA:
 - if not, check your `/etc/resolv.conf` file and modify it if necessary (give a correct DNS server)
 - use `ifconfig` to check your network configuration
 - check your firewall(s) and allow XOA to reach xen-orchestra.com (port 443)
+
+## XenServer Updates
+
+Starting September 2023 (XO ≥ 5.87), XenServer Updates require authentication:
```

Comment:
This seems to imply that XenServer updates require authentication since XO 5.87.

Reply:
...which is also pretty accurate in terms of chronology, but I know what you mean. Maybe I can just remove the XO version? The documentation always assumes that the latest version is installed anyway so in the end, I don't think it's necessary to mention an XO version at all.

Reply:
Much better, thank you.